<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SeerEditorWidgetForm</class>
 <widget class="QWidget" name="SeerEditorWidgetForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>838</width>
    <height>720</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SeerEditorWidgetForm</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QGridLayout" name="alternateBarLayout">
     <item row="1" column="3">
      <widget class="QToolButton" name="alternateAddToGlobalToolButton">
       <property name="toolTip">
        <string>Add path to global list of alternate directories.</string>
       </property>
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset resource="resource.qrc">
         <normaloff>:/seer/resources/RelaxLightIcons/list-add.svg</normaloff>:/seer/resources/RelaxLightIcons/list-add.svg</iconset>
       </property>
      </widget>
     </item>
     <item row="0" column="0" colspan="5">
      <widget class="QLabel" name="alternateLabel">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="1" column="4">
      <widget class="QToolButton" name="alternateCloseToolButton">
       <property name="toolTip">
        <string>Close the alternate bar.</string>
       </property>
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset resource="resource.qrc">
         <normaloff>:/seer/resources/RelaxLightIcons/list-remove.svg</normaloff>:/seer/resources/RelaxLightIcons/list-remove.svg</iconset>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="Line" name="line_3">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLineEdit" name="alternateLineEdit">
       <property name="toolTip">
        <string>Enter path for alternate location.</string>
       </property>
       <property name="placeholderText">
        <string>Enter /path/to/alternate/location</string>
       </property>
       <property name="clearButtonEnabled">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QToolButton" name="alternateFileOpenToolButton">
       <property name="toolTip">
        <string>Open file dialog for alternate location.</string>
       </property>
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset resource="resource.qrc">
         <normaloff>:/seer/resources/RelaxLightIcons/document-open.svg</normaloff>:/seer/resources/RelaxLightIcons/document-open.svg</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="SeerEditorWidgetSourceArea" name="sourceWidget" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>100</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="searchBarLayout">
     <item>
      <widget class="QLineEdit" name="searchTextLineEdit">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>80</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="toolTip">
        <string>Enter text to search for.</string>
       </property>
       <property name="placeholderText">
        <string>Enter search text</string>
       </property>
       <property name="clearButtonEnabled">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="matchCaseCheckBox">
       <property name="text">
        <string>Match Case</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="matchesLabel">
       <property name="text">
        <string></string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="searchDownToolButton">
       <property name="toolTip">
        <string>Search for the next occurance.</string>
       </property>
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset resource="resource.qrc">
         <normaloff>:/seer/resources/RelaxLightIcons/go-down.svg</normaloff>:/seer/resources/RelaxLightIcons/go-down.svg</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="searchUpToolButton">
       <property name="toolTip">
        <string>Search for the previous occurance.</string>
       </property>
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset resource="resource.qrc">
         <normaloff>:/seer/resources/RelaxLightIcons/go-up.svg</normaloff>:/seer/resources/RelaxLightIcons/go-up.svg</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="Line" name="line_1">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="searchLineNumberLineEdit">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>20</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="toolTip">
        <string>Scroll to a specific line.</string>
       </property>
       <property name="placeholderText">
        <string>Go to line #</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="Line" name="line_2">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="searchCloseToolButton">
       <property name="toolTip">
        <string>Close the search bar.</string>
       </property>
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset resource="resource.qrc">
         <normaloff>:/seer/resources/RelaxLightIcons/list-remove.svg</normaloff>:/seer/resources/RelaxLightIcons/list-remove.svg</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>SeerEditorWidgetSourceArea</class>
   <extends>QWidget</extends>
   <header location="global">SeerEditorWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="resource.qrc"/>
 </resources>
 <connections/>
</ui>
